.github/copilot-instructions.md

```diff
@@ -0,0 +1 @@
+For both new and existing code, help me enforce a maximum line length of 140 characters.
```

Comment:
Out of curiosity, what's the basis for 140?  I've seen 80, 120, and others but not 140.  When I searched for "What is a common maximum line length in code?" I got "Around 80 to 125 characters".

Reply:
I just used whatever looked good on my monitor lol. 125 can work too. Do you have a strong preference?

Reply:
No, I don't have a strong preference.  I think "whatever looks good on my monitor" will vary from user to user, but this is a good start.  Thanks, @aortiz-msft!